docs/build/isc/v1.1/docs/tutorials/defi-lend-borrow-tutorial-part-1.md

```diff
@@ -0,0 +1,86 @@
+# DeFi Lend Borrow
```

Comment:
This file has already been added to https://github.com/iotaledger/iota-wiki/pull/1675. If both parts I and II are ready, please add them in a single PR.

Reply:
as well, since it\'s a two part article, please indicate that this is part I